doc/src/suite-config.rst

```diff
@@ -0,0 +1,3772 @@
+.. _SuiteDefinition:
+
+Suite Configuration
+===================
+
+Cylc suites are defined in structured, validated, *suite.rc* files
+that concisely specify the properties of, and the relationships
+between, the various tasks managed by the suite. This section of the
+User Guide deals with the format and content of the suite.rc file,
+including task definition. Task implementation - what's required of the
+real commands, scripts, or programs that do the processing that the
+tasks represent - is covered in :ref:`TaskImplementation`; and
+task job submission - how tasks are submitted to run - is
+in :ref:`TaskJobSubmission`.
+
+
+.. _SuiteDefinitionDirectories:
+
+Suite Configuration Directories
+-------------------------------
+
+A cylc *suite configuration directory* contains:
+
+- **A suite.rc file**: this is the suite configuration.
+
+  - And any include-files used in it (see below; may be
+    kept in sub-directories).
+
+- **A ``bin/`` sub-directory** (optional)
```

Comment:
Annoyingly RST can't handle a literal inside bold text, other markup languages can but RST is somewhat deficient here.

This will find other examples (along with some false positives):

```console
$ git grep -E '\*[^\*]+`[^\*]+\*'
```

Reply:
Thanks, I'll go through & amend those instances.